monitor/grafana/zeebe.json

```diff
@@ -4497,21 +4611,27 @@
           "steppedLine": false,
           "targets": [
             {
-              "expr": "sum (jvm_buffer_pool_used_bytes{namespace=~\"$namespace\",pod=~\"$pod\"}) by (namespace,pod,pool)",
-              "format": "time_series",
-              "intervalFactor": 1,
-              "legendFormat": "Used {{pod}}:{{pool}}",
+              "expr": "sum(jvm_threads_current{namespace=~\"$namespace\", pod=~\"$pod\"}) by (namespace, pod)",
```

Comment:
:question: What we want to know is the latest value, no? If we use `sum`, would it sum the counts even after a restart, leading to an inflated number? Just wondering. Same below.

:thought_balloon: Or maybe it doesn't matter? You still get the right number in a point in time, right? Yeah `sum` is probably fine :smile:

Reply:
I think I can remove the sum